colin-api/src/colin_api/models/corp_party.py

```diff
@@ -22,7 +22,7 @@
 from flask import current_app
 
 from colin_api.exceptions import PartiesNotFoundException
-from colin_api.models import Address  # pylint: disable=cyclic-import
+from colin_api.models import Address, Business
 from colin_api.resources.db import DB
```

Comment:
move the DB before the models, as they import it.

Reply:
thanks, was wondering what to do here

Reply:
hmm it still doesn't like it so I'm going to put it back and add the pylint disable for now (it was there before - I don't remember why, but I took it off in my changes)

Reply:
alright changed it again haha - this was a tricky one because it wasn't coming up locally